### ISSUE TITLE
Repo contains a LIVE token publicly available

### ISSUE THREAD

**Issue by amriteya** [Manav1995/lunchBox#3]
The zomato token is hard coded in the repo and the repo is publicly available. Take it down.

**Comment by thelostone-mc** [comment]
Felt lazy passing it parameterized :P
Figured I'd get it done later at some point of time. 
Will get to it though!

**Comment by amriteya** [comment]
Yeah, it can be figured out later also, but the repo should not be publicly available till that token is there.

**Comment by thelostone-mc** [comment]
Token belongs @Manav1995. Closing it as he's alright with it being open for dev purposes.

**Comment by amriteya** [comment]
Token belongs to me.

![image](https://user-images.githubusercontent.com/4987564/29109214-e2ce7a1e-7cff-11e7-8036-7e52a6656aed.png)

**Comment by thelostone-mc** [comment]
@Manav1995 ?
